ratis-logservice/src/main/java/org/apache/ratis/logservice/server/LogStateMachine.java

```diff
@@ -331,7 +331,7 @@ public StateMachineStorage getStateMachineStorage() {
     LogServiceProtos.GetExportInfoReplyProto.Builder exportBuilder =
         LogServiceProtos.GetExportInfoReplyProto.newBuilder();
     exportMap.values().stream().map(
-        archivalInfo -> exportBuilder.addInfo(LogServiceProtoUtil.toExportInfoProto(archivalInfo)))
+        archInfo -> exportBuilder.addInfo(LogServiceProtoUtil.toExportInfoProto(archInfo)))
```

Comment:
What's the checkstyle rule this is failing on?

Reply:
{{archivalInfo}} is hiding another private member of the class